test/Database/LSMTree/Model/Normal/Session.hs

```diff
@@ -308,17 +308,33 @@ deletes ::
   -> m ()
 deletes = updates . fmap (,Model.Delete)
 
+data BlobRef blob = BlobRef {
+    parentTable :: forall k v. TableHandle k v blob
```

Comment:
The specification isn't clear. Are blobs table-specific or shared across the session? What if I `duplicate` and close other table? Will the blobrefs in duplicated table be different (I doubt?)

Reply:
See the haddock comment on `BlobRef`

Reply:
Perhaps have the haddock comment refer back there for details.